operational_analysis/methods/plant_analysis.py

```diff
@@ -850,11 +856,20 @@ def run_AEP_monte_carlo(self):
             # Get long-term availability and curtailment losses by month
             [avail_lt_losses, curt_lt_losses] = self.sample_long_term_losses()  
 
-            # Assign AEP, long-term availability, and long-term curtailment to output data frame
+            # Assign AEP, IAV, long-term availability, and long-term curtailment to output data frame
             aep_GWh[n] = gross_lt.sum()/self._run.num_years_windiness * (1 - avail_lt_losses)
+            iav[n] = gross_lt_annual.std()/gross_lt_annual.mean()
             avail_pct[n] = avail_lt_losses
             curt_pct[n] = curt_lt_losses
             lt_por_ratio[n] = (gross_lt.sum()/self._run.num_years_windiness) / gross_por.sum()
+
+        # Calculate mean IAV for gross energy
+        iav_avg = iav.mean()
+        
+        # Apply IAV to AEP from single MC iterations
+        iav_nsim = np.random.normal(1, iav_avg, self.num_sim)
+        aep_GWh = aep_GWh * iav_nsim
+        lt_por_ratio = lt_por_ratio * iav_nsim
             
         # Return final output
         sim_results = pd.DataFrame(index=np.arange(num_sim), data={'aep_GWh': aep_GWh,                                                                                                        
```

Comment:
It might make sense to save the `iav` array in the sim_results. What do you think?

Reply:
Yes, I like that idea. It's now incorporated.